test_user_management.py

```
@@ -0,0 +1,391 @@
+        return False
+    
+    def hash_password(self, password: str) -> str:
+        """Hash password using MD5 (SECURITY ISSUE 10: Weak hashing)"""
+        return hashlib.md5(password.encode()).hexdigest()
+    
+    def generate_session_token(self, user_id: int) -> str:
+        """Generate session token"""
+        # SECURITY ISSUE 11: Weak token generation
+        import time
+        token = f"{user_id}_{int(time.time())}_{SECRET_KEY}"
+        return hashlib.md5(token.encode()).hexdigest()
```

Ruff (0.15.6)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)